src/Microsoft.Health.Dicom.Web/DevelopmentIdentityProviderRegistrationExtensions.cs

```diff
@@ -0,0 +1,238 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using System.Text.RegularExpressions;
+using EnsureThat;
+using IdentityServer4.Models;
+using IdentityServer4.Test;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+
+namespace Microsoft.Health.Dicom.Web
+{
+    public static class DevelopmentIdentityProviderRegistrationExtensions
```

Comment:
I am assuming this all goes to the share health.development.identity

Reply:
Yes, a large portion of this can be abstracted to shared components.

Reply:
I already have initial authentication. Will this move in the next PR?